vagrant/tutorial/Vagrantfile

```diff
@@ -29,13 +29,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
                         ENV.fetch('FLOCKER_BUILD_SERVER', '')
                       ]
 
-  config.vm.provision 'docker', images: [
-    'busybox',
-    'clusterhq/mongodb',
-    'redis',
-    'python:2.7-slim',
-    'clusterhq/flask'
-  ]
+  config.vm.provision :shell,
```

Comment:
Perhaps worth some commentary that the Vagrant Docker provisioner is fragile and we're intentionally not using it even though it's obviously the correct thing to use.